packages/home-spa/src/js/controller.js

```diff
@@ -1,62 +1,75 @@
 // Services
-import { spas, microservices, carouselList } from './service.js';
+import { getData }  from './service.js';
+import '../data.json';
 
 const microserviceCard = document.getElementById('microservice-cards');
 
```

Comment:
Why the blank lines between const statements?

Reply:
removed blank lines